package.json

```diff
@@ -37,14 +37,9 @@
   "devDependencies": {
     "@gravitational/build": "workspace:*",
     "@ianvs/prettier-plugin-sort-imports": "^4.4.1",
-    "@storybook/addon-actions": "^8.6.12",
-    "@storybook/addon-controls": "^8.6.12",
-    "@storybook/addon-toolbars": "^8.6.12",
-    "@storybook/components": "^8.6.12",
-    "@storybook/preview-api": "^8.6.12",
-    "@storybook/react": "^8.6.12",
-    "@storybook/react-vite": "^8.6.12",
-    "@storybook/test-runner": "^0.22.0",
+    "@storybook/react": "^9.0.15",
```

Comment:
Did you check out the upgrade command from the migration guide? https://storybook.js.org/docs/migration-guide

I just ran it on master and it seems to includes much more changes than are included in this PR, e.g., it replaces `@storybook/react` with `@storybook/react-vite`, doesn't seems to bring in addon-vitest, and attempts to add an ESLint config – it does so incorrectly, but I think it'd be worthwhile to enable it since the recommended one doesn't seem to report any issues at the moment.